src/yb/master/backfill_index.cc

```diff
@@ -653,7 +653,8 @@ BackfillTable::BackfillTable(
     if (!first) {
       out << ", ";
     }
-    out << master_->catalog_manager()->GetTableInfo(index_id)->name();
```

Comment:
I think we can revert this part. Having just the names should be fine for displaying information.

(ids may make it too cluttered.)